src/less/less/ds6/variables.less

```diff
@@ -394,6 +391,9 @@
 @ds6-icon-inline-notice-confirmation-base64: @ds6-icon-confirmation-base64;
 @ds6-icon-inline-notice-information-base64: @ds6-icon-information-base64;
 @ds6-icon-inline-notice-priority-base64: @ds6-icon-inline-notice-attention-base64;
+@ds6-icon-page-notice-attention-base64: @ds6-icon-attention-base64;
+@ds6-icon-page-notice-confirmation-base64: @ds6-icon-confirmation-base64;
+@ds6-icon-page-notice-information-base64: @ds6-icon-information-base64;
```

Comment:
Are we using these anymore? Are you keeping them around simply for those who might have used them, to keep from having breaking changes? If so, should we mark the as deprecated?

Reply:
Yep, this whole group of aliases is marked as deprecated on line 387.

Reply:
Ah, thanks!